plugin/__init__.py

```diff
@@ -468,10 +468,15 @@ def removeDeckConfigId(self, configId):
 
 
     @util.api()
-    def storeMediaFile(self, filename, data=None, url=None):
+    def storeMediaFile(self, filename, data=None, file=None, url=None):
         if data:
             self.deleteMediaFile(filename)
             self.media().writeData(filename, base64.b64decode(data))
+        elif file:
+            self.deleteMediaFile(filename)
+            f = open(file, 'rb')
```

Comment:
Please change to something like:
```
with open(file, 'rb') as fp:
    self.media().writeData(filename, f.read())
```
This will ensure that the file is properly closed.